presto-docs/src/main/sphinx/sql/revoke.rst

```diff
@@ -0,0 +1,53 @@
+======
+REVOKE
+======
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    REVOKE [ GRANT OPTION FOR ]
+    (privilege [, ...] | ALL PRIVILEGES)
+    ON [ TABLE ] table_name FROM grantee
+
+Usage of the term ``grantee`` denotes both, users and roles.
+
+Description
+-----------
+
+Revokes the specified privileges from the specified grantee.
+
+Specifying ``ALL PRIVILEGES`` revokes ``DELETE, INSERT`` and ``SELECT`` privileges.
+
+Specifying ``PUBLIC`` revokes privileges from all grantees.
```

Comment:
Seems a little extreme.  Is there an admin user who can reverse that, or would it be necessary to drop and re-create the table to restore access?  Either way, we should make this explicit in the documentation.